### ISSUE TITLE
Teams `/start @user`

### ISSUE THREAD

**Issue by 0x4007** [ubiquibot/command-start-stop#6]
This reminds me that it would be really interesting in the future to support teams. Where I could write on a new task 

```
/start @Keyrxng 
```

And we would both be assigned (and be eligible to split the assignee reward.)

I know a lot of developers work as lone rangers but it could make the experience more collaborative and exciting if we have a massive tasks marketplace and teams hunt projects to co-op on together.

Also if I am already assigned to a project and I write the above command, then it will add them as an assignee as well.

**Comment by Keyrxng** [comment]
So I could start an issue as a team by saying `/start @0x4007` and it'll assign us both.

Is this "team" now considered a single entity or two singles? E.g, could I then `/stop` and it'll remove my assignment but you would keep yours or we're both removed?

<hr/>

Only one person is going to own the PR which makes `/stop` with auto-close for PRs tricky.

I think to remove any burden from the user we:
1. check if multiple assignees exist
2. If they do and the person quitting is the PR owner then we do not close the PR, on the off chance that the other person is carrying on the work involved.
3. If there is no activity on the issue then the remaining assignee should be kicked and the PR closed?

otherwise, the person still working will need to re-open the PR (which isn't a big deal but not the best UX)

**Comment by 0x4007** [comment]
I suppose just check if the o o person quitting is the pull author. If so then close it. If not then don't.

**Comment by Keyrxng** [comment]
> Warning!	This task was created over 13 days ago. Please confirm that this issue specification is accurate before starting.
> Deadline	Mon, Jul 15, 8:15 PM UTC
> Registered Wallet	0xAe5D1F192013db889b1e2115A370aB133f359765
> ###### Tips:
> * Use `/wallet 0x0000...0000` if you want to update your registered payment wallet address.
> * Be sure to open a draft pull request as soon as possible to communicate updates on your progress.
> * Be sure to provide timely updates to us when requested, or you will be automatically unassigned from the task.

Does this assignment comment need updated if it's this version of the command that's used?

**Comment by 0x4007** [comment]
> Does this assignment comment need updated if it's this version of the command that's used?

Let's handle it in another task if needed! Not sure if necessary, will know after testing the feature some time in the future.

**Comment by 0x4007** [comment]
> Does this assignment comment need updated if it's this version of the command that's used?

Let's handle it in another task if needed! Not sure if necessary, will know after testing the feature some time in the future.

**Comment by 0x4007** [comment]
@Keyrxng I don't wanna forget about installing this feature so after you get it installed mark it as complete

**Comment by 0x4007** [comment]
@Keyrxng can you install this plugin on the config

**Comment by Keyrxng** [comment]
@0x4007 it doesn't require any sort of enabling it's just built-in so if the worker that's deployed for `start-stop-command` is up to date then it should be working right now

@gentlementlegen can you confirm, pretty sure it's active right now?

**Comment by 0x4007** [comment]
You guys need to focus on less tasks and get them done, probably!

**Comment by gentlementlegen** [comment]
@0x4007 I think the max is 3 currently, regardless of collaborator / contributor so pretty easy to hit (I have 3 huge tasks ongoing for me).
Also installing this feature just means merging into `main` to deploy the latest version of the worker.

**Comment by 0x4007** [comment]
Time to lock in on three or less and complete them!

**Comment by Keyrxng** [comment]
@0x4007 are you happy to close this now aye?
